site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -261,6 +265,11 @@ Whether a parent email is required.
 [field]#tenant.familyConfiguration.parentRegistrationEmailTemplateId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.8.0#::
 The unique Id of the email template to use for parent registration.
 
+[field]#tenant.formConfiguration.adminUserFormId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.20.0#::
+The unique Id of the form to use for the Add and Edit User form when used in the FusionAuth admin UI. 
++
+A paid edition of FusionAuth is required to utilize custom forms. When this parameter is not provided it will default to a form provided by FusionAuth that may be used without a paid edition.
```

Comment:
See other examples of describing the default in the description, there are some style patterns. This value has a specific default, it will be the form assigned to the default tenant, you can review the code path to see how the default is configured.